src/app/forms/validators/custom-validators.ts

```diff
@@ -114,6 +114,15 @@ export class CustomValidators {
     };
   };
 
+  static defined = (): ValidatorFn => {
+    return (control: AbstractControl): ValidationErrors | null => {
+      if (typeof control.value === 'undefined') {
```

Comment:
Why not?
```suggestion
      if (control.value === undefined) {
```

Reply:
I read that `typeof` should be used in the case that the variable could potentially not be declared (it's safer), otherwise you would get a reference error by doing a strict `control.value === undefined` check.